content/index.pl.md

```diff
@@ -0,0 +1,144 @@
+
+Stanowi on podstawę do zrozumienia różnych standardów dostępności opracowanych przez Inicjatywę na Rzecz Dostępności Internetowej (WAI), która jest częścią Konsorcjum <abbr title="World Wide Web">W3C</abbr>.
+
+{::nomarkdown}
+{% include box.html type="end" %}
+{:/}
+
+{::nomarkdown}
+{% include_cached toc.html type="start" title="Treść strony" class="simple" %}
+{:/}
+
+{::options toc_levels="2" /}
+
+-   This text will be replaced by the TOC.
+{:toc}
+
+{::nomarkdown}
+{% include_cached toc.html type="end" %}
+{:/}
+
+
+## Wprowadzenie {#intro}
+{:.no_toc}
+
+Aby Internet był dostępny dla osób z&nbsp;niepełnosprawnościami, konieczne jest współdziałanie kilku różnych komponentów związanych z&nbsp;projektowaniem i&nbsp;tworzeniem treści cyfrowych. Te komponenty to:
+
+-   **treść** - informacje na stronie internetowej lub w aplikacji internetowej, w tym:
+    -   informacje naturalne (takie jak tekst, obraz i dźwięki) oraz
+    -   kod lub znaczniki definiujące strukturę, prezentację itp.
+-   **przeglądarki internetowe, odtwarzacze multimedialne**, i inne „programy użytkownika”
+-   **technologie wspomagające**, w niektórych przypadkach - czytniki ekranu, alternatywne klawiatury, przełączniki, oprogramowanie skanujące itp.
+-   **użytkownicy**, ich wiedza, doświadczenia, a w niektórych przypadkach, sposób korzystania z internetu (strategie adaptacyjne
+-   **twórcy** - projektanci, programiści, autorzy itd., w tym twórcy z&nbsp;niepełnosprawnościami i&nbsp;użytkownicy, którzy współtworzą treść
+-   **narzędzia autorskie** - oprogramowanie do tworzenia treści cyfrowych
+-   **narzędzia oceniające** - oprogramowanie do oceny dostępności cyfrowej, walidatory HTML i CSS itp.
+
+## Jak komponenty są ze sobą powiązane {#relate}
+
+{% assign longdesc_url = "/fundamentals/components/examples/pl#relate" | relative_url %}
+{% include image.html src="relate.pl.png" alt="Ilustruje, jak komponenty są powiązanie" longdesc=longdesc_url %}
+
+**Twórcy** stron internetowych zwykle używają **narzędzi autorskich** i narzędzi oceniających do tworzenia **treści** cyfrowych.
+
+**Ludzie** („**użytkownicy**”) używają **przeglądarek, odtwarzaczy mediów, technologii wspomagających** lub innych „**programów użytkownika**”, aby uzyskać dostęp do treści cyfrowych i&nbsp;wchodzić z&nbsp;nimi w&nbsp;interakcje.
+
+## Współzależności komponentów {#depend}
+
+Między tymi komponentami istnieją istotne współzależności; to znaczy że komponenty muszą ze sobą współpracować, aby Internet był dostępny. Na przykład, dla tekstu alternatywnego opisującego obraz:
+
+-   **specyfikacja techniczna** wymaga alternatywy tekstowej (np. HTML definiuje atrybut tekstu alternatywnego (`alt`) w elemencie obrazka (`img`)
+-   **wytyczne WAI** - WCAG, ATAG i UAAG, wymienione w dalszej części - określają, jak wdrożyć tekst alternatywny w&nbsp;różnych komponentach, aby zwiększyć dostępność
```

Comment:
```suggestion
-   **wytyczne WAI** - WCAG, ATAG i UAAG, wymienione w dalszej części - określają, jak wdrożyć tekst alternatywny w&nbsp;różnych komponentach, aby zwiększyć dostępność
```

A link is missing here. Original version is `([WCAG, ATAG, UAAG described below](#guidelines))`